Specs/DataSources/PolylineGeometryUpdaterSpec.js

```diff
@@ -561,6 +588,58 @@ defineSuite([
         updater.destroy();
     });
 
+    it('lineType can be dynamic', function() {
+        if (!Entity.supportsPolylinesOnTerrain(scene)) {
+            return;
+        }
+
+        var entity = new Entity();
+        var polyline = new PolylineGraphics();
+        entity.polyline = polyline;
+
+        var time = new JulianDate(0, 0);
+
+        var lineTypeVar = LineType.GEODESIC;
+        var lineType = new CallbackProperty(function() {
+            return lineTypeVar;
+        }, false);
+
+        polyline.show = new ConstantProperty(true);
+        polyline.width = new ConstantProperty(1.0);
+        polyline.positions = new ConstantProperty([Cartesian3.fromDegrees(0, 0, 0), Cartesian3.fromDegrees(0, 1, 0)]);
+        polyline.material = new ColorMaterialProperty(Color.RED);
+        polyline.followSurface = new ConstantProperty(true);
+        polyline.granularity = new ConstantProperty(0.001);
+        polyline.clampToGround = new ConstantProperty(true);
```

Comment:
It doesn't look like this spec necessarily has to be `clampToGround`, it might be slightly faster if it isn't clamped to ground.